### PR TITLE
Added a new line in a READ ME.

### PR DESCRIPTION
**Pull request by pasang2044**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,3 +55,6 @@ The following command will open the door if the door is locked.
 
 The following command will give you a full description of your location.
 *	GPS
+
+
+More Instructions as we go along.
```
Comment (SethM-SDE) [inline README.md]: Why is this here?

**Review by SethM-SDE — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (pasang2044) [reply README.md]: I deleted it.

**Review by SethM-SDE — APPROVED** [review]
